libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
Thank you your comments.

In IAR compiler, "__disable_irq()" of intrinsic function have no return value.
Therefore, when we need to use the return value of "__disable_irq()", we create "__disable_irq_iar ()" and use it.